ui/src/file.mock.ts

```diff
@@ -0,0 +1 @@
+module.exports = {};
```

Comment:
Is there a reason we need this?

Reply:
Not sure. It was there when I started working on the branch.